index.js

```diff
@@ -212,6 +212,7 @@
       case 1: return function(a) { return f.apply(this, arguments); };
       case 2: return function(a, b) { return f.apply(this, arguments); };
       case 3: return function(a, b, c) { return f.apply(this, arguments); };
+      case 4: return function(a, b, c, d) { return f.apply(this, arguments); };
```

Comment:
If you rebase, this line should disappear from the diff, since it was added in one of your other pull requests. :)